docs/opencompetition2021.md

```diff
@@ -0,0 +1,48 @@
+#  Open Optimization Competition 2021: Competition and Benchmarking of Sampling-Based Optimization Algorithms
```

Comment:
Can you make a rst file instead (requires some different formatting though), and reference it in the documention website?